cstar/roms/simulation.py

```diff
@@ -979,10 +1003,9 @@ def to_blueprint(self, filename: str | Path) -> None:
         from_blueprint : Creates a `ROMSSimulation` instance from a YAML blueprint.
         to_dict : Converts the instance into a dictionary representation.
         """
-        with open(filename, "w") as yaml_file:
-            yaml.dump(
-                self.to_dict(), yaml_file, default_flow_style=False, sort_keys=False
-            )
+        raise NotImplementedError(
```

Comment:
Consider updating the docstring, too. Forcing the user to run the code and let it break feels funny. If they see `THIS METHOD HAS BEEN DEPRECATED` in the docstring, they may avoid it.

Python 3.13 introduces `typing.deprecated`, so someday we'll be able to solve more elegantly.